e2e/testconfig/testconfig.go

```diff
@@ -362,8 +362,6 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string, cometCf
 	tmTomlOverrides["log_level"] = cometCfg.LogLevel // change to debug in ~/.ibc-go-e2e-config.json to increase cometbft logging.
 	configFileOverrides["config/config.toml"] = tmTomlOverrides
 
-	useNewGenesisCommand := cc.Binary == icadBinary && testvalues.IcadNewGenesisCommandsFeatureReleases.IsSupported(cc.Tag)
```

Comment:
You can now delete [`IcadNewGenesisCommandsFeatureReleases` from `values.go`](https://github.com/cosmos/ibc-go/blob/main/e2e/testvalues/values.go#L63), since it is not used anywhere else.

Reply:
Is `icad` wired up to use this new genesis cmd structure also? is hardcoding to `true` below ok for compatibility test runs?

Reply:
oh, yes, you're right, @damiannolan! We cannot remove this yet, because previous version of icad don't have the genesis subcommand. Good catch!

Reply:
ACK once this is added back